adb_graphics/specs.py

```diff
@@ -197,7 +197,24 @@ def lcl_colors(self) -> np.ndarray:
         ctable = ctables.colortables.get_colortable(self.vspec.get('cmap')) \
                     (range(60, 180, 6)) # rainbow
         return ctable
+    @property
```

Comment:
Please add white space between each new function above the `@property` decorator.